src/components/common/SmallTile/SmallTile.jsx

```diff
@@ -1,46 +1,48 @@
 import { Genres } from "./Genres/Genres";
 import {
-	Content,
-	TileHeader,
-	IconStar,
-	Image,
-	Rating,
-	Rate,
-	Votes,
-	Year,
-	Wrapper,
-	ImageWrapper,
-	StyledPictureIcon,
-	MovieInfo
+ Content,
+ TileHeader,
+ IconStar,
+ Image,
+ Rating,
+ Rate,
+ Votes,
+ Year,
+ Wrapper,
+ ImageWrapper,
+ StyledVideoIcon,
+ MovieInfo
 } from "./styled";
 
 export const SmallTile = ({ movie }) => {
-	return (
-		<Wrapper>
-			<ImageWrapper>
-				{movie.poster_path ? (
-					<Image
-						src={`https://image.tmdb.org/t/p/original${movie.poster_path}`}
-						alt=""
-					/>
-				) : (
-					<StyledPictureIcon />
-				)}
-			</ImageWrapper>
-			<Content>
-				<MovieInfo>
-					<TileHeader>{movie.title}</TileHeader>
-					<Year>{movie.release_date.slice(0, 4)}</Year>
-					<Genres genreIds={movie.genre_ids} />
-				</MovieInfo>
-				<Rating>
-					<IconStar />
-					<Rate>{movie.vote_average.toFixed(1)}</Rate>
-					<Votes>
-						{movie.vote_count > 0 ? `${movie.vote_count} votes` : "No votes"}
-					</Votes>
-				</Rating>
-			</Content>
-		</Wrapper>
-	);
+ return (
+  <Wrapper>
+   <ImageWrapper>
+    {movie.poster_path ? (
+     <Image
+      src={`https://image.tmdb.org/t/p/original${movie.poster_path}`}
+      alt=""
+     />
+    ) : (
+     <StyledVideoIcon />
+    )}
+   </ImageWrapper>
+   <Content>
+    <MovieInfo>
+     <TileHeader>{movie.title}</TileHeader>
+     <Year>{movie.release_date.slice(0, 4)}</Year>
+     <Genres genreIds={movie.genre_ids} />
+    </MovieInfo>
+    {movie.vote_count > 0 ? (
+     <Rating>
+      <IconStar />
+      <Rate>{movie.vote_average.toFixed(1)}</Rate>
+      <Votes>{`${movie.vote_count} votes`}</Votes>
+     </Rating>
+    ) : (
+     <Votes>No votes</Votes>
```

Comment:
suggest adding "yet" at this point as in the project
<img width="168" alt="Zrzut ekranu 2023-12-1 o 13 00 52" src="https://github.com/jakub-bartoszek-developer/Movie-browser/assets/131645596/bc5b6756-b87a-4c1d-a09c-8afd52017356">